ansible_rulebook/websocket.py

```diff
@@ -34,6 +34,7 @@
 from ansible_rulebook.vault import Vault, has_vaulted_str
 
 logger = logging.getLogger(__name__)
+logging.getLogger("websockets").setLevel(logging.ERROR)
```

Comment:
The problem is the "DEBUG" level, right? Should we honor the `-vv` and set the INFO level instead of forcing always the ERROR one?

Reply:
@Alex-Izquierdo That's when it leaks from the websocket library logging every HTTP byte. We never want that level of tracing from the websocket layer we only need errors from it.